src/index.js

```diff
@@ -0,0 +1,197 @@
+/***********************************************************
+            Temperature variables and controls
+***********************************************************/
+
+const tempDisplay = document.getElementById("temp-display");
+let temperature = parseInt(tempDisplay.textContent);
+
+const updateTemperature = () => {
```

Comment:
Excellent! It looks like the color decisions are based on if the temperature is in Celsius. Consider what could be refactored so that the color works well for Fahrenheit too.